spec.html

```diff
@@ -37915,10 +37956,16 @@ <h1>Atomics.store ( _typedArray_, _index_, _value_ )</h1>
 
     <emu-clause id="sec-atomics.sub">
       <h1>Atomics.sub ( _typedArray_, _index_, _value_ )</h1>
-      <p>Let `subtract` denote a semantic function of two List of byte values arguments that applies the subtraction operation to the Number values corresponding to the List of byte values arguments and returns a List of byte values corresponding to the result of that operation.</p>
       <p>The following steps are taken:</p>
       <emu-alg>
-        1. Return ? AtomicReadModifyWrite(_typedArray_, _index_, _value_, `subtract`).
+        1. Let _subtract_ be a new read-modify-write modification function with parameters (_xBytes_, _yBytes_) that captures _type_ and _isLittleEndian_ and performs the following steps atomically when called:
```

Comment:
This presumably needs the same

```
        1. Let _type_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _typedArray_.[[TypedArrayName]].
        1. Let _isLittleEndian_ be the value of the [[LittleEndian]] field of the surrounding agent's Agent Record.
```
steps that `Atomics.add` has.

Reply:
Ack.